### ISSUE TITLE
crossvent.js?

### ISSUE THREAD

**Issue by samcov** [bevacqua/dragula#18]
Ok, I am loading in the head, but what is crossvent.js?

I can't even find it on my system, but dragula is dying there in "function addEventEasy"

Color me confused...

**Comment by samcov** [comment]
Ok, I figured out that dragula isn't looking for the id's of the elements to be containers, it wants the object itself, so once I passed that, I got the error that everyone has when they load it in the head section.

Then I loaded it just before the closing body tag, and my complex UI worked perfectly... Yaaa!

While it is no problem for me to load just before the closing body tag, it would much cleaner if I could load it in the head section with everything else to take advantage of script bundling!

It's likely that you're waiting until the user interface is settled, and jquery handles this elegantly doing initializations in the .ready method, but as I said, I'm good with the way it is!

**Fantastic Library**, I love it!!!
